tailwind.config.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
       fontFamily: {
         number: ['BioRhyme', 'sans-serif'],
         sans: ['Montserrat Sans', ...defaultTheme.fontFamily.sans]
-      }
+      },
+      colors: {
+        'custom-color': '#367055',
```

Comment:
Issue: The label `custom-color` isn't meaningful, all it tells us is that it's not from Tailwind. Could we call this something like `brand-primary` instead?